subsys/bluetooth/controller/ll_sw/nordic/lll/lll_adv_pdu.h

```diff
@@ -65,6 +65,28 @@ static inline struct pdu_adv *lll_adv_scan_rsp_peek(struct lll_adv *lll)
 	return (void *)lll->scan_rsp.pdu[lll->scan_rsp.last];
 }
 
+static inline struct pdu_adv *
+lll_adv_pdu_latest_peek(struct lll_adv_pdu const *const pdu)
```

Comment:
Isn't it more typical to have `const` as the first variable modifier, e.g. 
```suggestion
lll_adv_pdu_latest_peek(const struct lll_adv_pdu *const pdu)
```

This comment applies multiple places.

Reply:
Both coding styles are permitted.

Reply:
They are, but I was just wondering if the suggestion from me is more in line with most other occurrences of `const` for the sake of being consistent in the code base.

Reply:
> This comment applies multiple places.

This change applies repo wide, hence should be taken up in separate PR and with wider audience.

Reply:
> > This comment applies multiple places.
> 
> This change applies repo wide, hence should be taken up in separate PR and with wider audience.

Does that mean that the change you are introducing here in this PR should be moved to another PR, or are you suggesting that any revert of this would be done in a separate PR?

Reply:
I have moved out the `const` qualifier commit. Please re-review.